kube/pod.go

```diff
@@ -139,7 +139,11 @@ func NewPodTemplate(role *model.Role, settings ExportSettings, grapher util.Mode
 	spec.Sort()
 
 	podTemplate := helm.NewMapping()
-	podTemplate.Add("metadata", newObjectMeta(role.Name))
+	meta := newObjectMeta(role.Name)
+	if settings.CreateHelmChart {
+		meta.Add("annotations", helm.NewMapping("checksum/config", `{{ include (print $.Template.BasePath "/secrets.yaml") . | sha256sum }}`))
```

Comment:
Shouldn't this include the generation number / generated secrets, so that pods get re-rolled when you just bump the generation?